cmd/cstor-volume-mgmt/app/command/commands.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2018 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openebs/maya/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdName = "cstor-volume-mgmt"
+	usage   = fmt.Sprintf("%s", cmdName)
```

Comment:
Why are we not doing simple assignment?

Reply:
Code copied from Pool watcher code. I guess it is good to have the names in constants like cmdName = "<something>". It would be easy to change the command name later on if at all we need to. AFAIK, it should not be a problem.